src/ddsrt/src/events.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright(c) 2006 to 2020 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+#include "dds/ddsrt/events.h"
+#include "dds/ddsrt/heap.h"
+
+dds_return_t ddsrt_event_socket_init(ddsrt_event_t* ev, ddsrt_socket_t sock, uint32_t flags) {
```

Comment:
For consistency with the rest of the code base, I'd prefer the opening brace that is part of function declaration syntax to (generally) sit on a line of its own. The "(generally)" is there because it is such a waste of space for trivial one-line or empty functions.

Reply:
I will change the braces on function declarations to match the rest of the code base.

Reply:
This is now implemented.